photon-api/src/main/scala/com/linkedin/photon/ml/function/glm/DistributedGLMLossFunction.scala

```diff
@@ -160,17 +160,17 @@ object DistributedGLMLossFunction {
   def apply(
       configuration: GLMOptimizationConfiguration,
       singleLossFunction: PointwiseLossFunction,
-      treeAggregateDepth: Int): DistributedGLMLossFunction = {
+      treeAggregateDepth: Int,
+      index: Int = -1): DistributedGLMLossFunction = {
```

Comment:
Can you add a comment for this parameter?

Reply:
In addition, parameter name and type should change (see below):
```
interceptIndexOpt: Option[Int] = None
```